code/index.html

```diff
@@ -1,30 +1,136 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <link rel="preconnect" href="https://fonts.gstatic.com"> 
+    <link href="https://fonts.googleapis.com/css2?family=Kaushan+Script&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com"> 
+    <link href="https://fonts.googleapis.com/css2?family=Kaushan+Script&family=Open+Sans&family=Oswald&display=swap" rel="stylesheet">
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>Penguin lifestyle</title>
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+
+    <header class="header">
+      <picture>
+        <source srcset="./images/emperor-penguins-mobile.png" media="(max-width 767px)">
+        <source srcset="./images/emperor-penguins-tablet.png" media=" (max-width 1999px)">
+        <img class="hero-image" src="./images/emperor-penguins.png" alt="picture of emperor penguins"/>
+      </picture>
+      <div class="title">
+      <h1 class="hero-title">PENGUIN LIFESTYLE</h1>
+      <h2 class="hero-subtitle">Learn how to live like a penguin in just 30 days!</h2>
+    </div>
+    </header>
```

Comment:
Remember to indent properly as it will make your code easier to read:
```
<div class="title">
  <h1 class="hero-title">PENGUIN LIFESTYLE</h1>
  <h2 class="hero-subtitle">Learn how to live like a penguin in just 30 days!</h2>
</div>
```

Reply:
Love your theme btw 😄